CMSgov/easi-app
# [NOREF] TRB attendee icon colors

## Changes and Description

- Updated `<InitialsIcon>` component to have different background colors
  - If `index` prop is provided, component uses corresponding background color from array
  - If `index` prop is undefined, uses `backgroundColor` prop OR defaults to `bg-primary-lighter`
-  Updated icons in `<Attendees>` component in TRB form to have different background colors
- Unit tests



## How to test this change

- Create TRB request form and go to Attendees section
- Add multiple attendees
- Confirm that attendee icons have different background colors

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
